### DISCUSSION TITLE
Color change for .show-10-secondary-axes

### DISCUSSION THREAD

**Discussion by sugimotosatoko** [ChartsCSS/charts.css#129] [Q&A] [answered]
About .show-10-secondary-axes, is it possible to change the color of only certain lines?
For example, I want only the fifth and tenth lines to be black.

**Comment by ramiy** [comment, marked as answer]
@sugimotosatoko

Yes, it's possible with the `:nth-of-type()` CSS pseudo-class.

To stye the fifth and tenth lines:

```css
#example tr:nth-of-type(5n) td {
  /* ... */
}
```

Read more: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type
